### PR TITLE
DOCSP-20626 combine help pages

### PR DESCRIPTION
**Pull request by davemungo**
JIRA https://jira.mongodb.org/browse/DOCSP-20626     
[MONGOSH] Combine Help pages

STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-20626-combine-help-pages/reference/access-mdb-shell-help/

### CODE REVIEW

File: source/reference/access-mdb-shell-help.txt [hunk 1]
```diff
@@ -12,238 +13,292 @@ Access Help in ``mongosh``
...
+.. code-block:: javascript
 
-       db.adminCommand({ serverStatus: 1 }):
+   db.adminCommand.help()
 
-       Runs an arbitrary command against the admin database.
+The output resembles the following:
 
-       For more information on usage: https://docs.mongodb.com/manual/reference/method/db.adminCommand
+.. code-block:: none
```
Comment (jason-price-mongodb) [inline source/reference/access-mdb-shell-help.txt:122]: javascript

File: source/reference/access-mdb-shell-help.txt [hunk 2]
```diff
@@ -12,238 +13,292 @@ Access Help in ``mongosh``
...
-  
-Show List of Collection Methods
-  To see the list of methods available on the collection objects
-  (e.g. ``db.<collection>``), use the ``db.<collection>.help()``
-  method:
+.. code-block:: javascript
+
+   db.collection.help()
+
+``<collection>`` can be the name of an existing or nonexistent
```
Comment (jason-price-mongodb) [inline source/reference/access-mdb-shell-help.txt:195]: non-existent

File: source/reference/access-mdb-shell-help.txt [hunk 3]
```diff
@@ -12,238 +13,292 @@ Access Help in ``mongosh``
...
-       returnType: { type: 'unknown', attributes: {} },
-       platforms: [ 0, 1, 2 ],
-       help: [Function] Help
-     }
+Cursor Help
+-----------
 
-.. didn't include the cursor details from the manual help page as
-   it seems as if most of it doesn't apply for ``mongosh``
+Use :ref:`cursor methods <mongosh-cursor-methods>` to modify
```
Comment (jason-price-mongodb) [inline source/reference/access-mdb-shell-help.txt:274]: "To XXX, use YYY."

File: source/reference/access-mdb-shell-help.txt [hunk 4]
```diff
@@ -308,58 +363,27 @@ certain database commands and obtain information on your deployment:
...
-       the current database.
-
-   * - ``show profile``
-
-     - Print the five most recent operations that took 1 millisecond or
-       more. See documentation on the
-       :manual:`database profiler
-       </tutorial/manage-the-database-profiler>` for more information.
+   * - ``show dbs``
+     - Print a list of all databases on the server.
```
Comment (jason-price-mongodb) [inline source/reference/access-mdb-shell-help.txt:380]: I would not use "Print". I would use "Display" or "Returns".

Same comment throughout.

File: source/reference/access-mdb-shell-help.txt [hunk 5]
```diff
@@ -308,58 +363,27 @@ certain database commands and obtain information on your deployment:
...
 
-   * - ``show logs``
+       .. note::
 
-     - Print the accessible logger names. See :doc:`/logs`.
+          ``show dbs`` is synonymous with ``show databases``.
```
Comment (jason-price-mongodb) [inline source/reference/access-mdb-shell-help.txt:384]: Change "is synonymous with" to "is the same as".

File: source/reference/access-mdb-shell-help.txt [hunk 6]
```diff
@@ -12,238 +13,292 @@ Access Help in ``mongosh``
    :depth: 1
    :class: singlecol
 
-``mongosh`` provides additional help information in its help system.
-This document provides an overview of accessing this help
-information.
+This document provides a brief overview of the help available in
```
Comment (jason-price-mongodb) [inline source/reference/access-mdb-shell-help.txt:16]: "an overview"

**Review by jason-price-mongodb — APPROVED** [review]
@davemungo LGTM with suggestions. Cheers, Jason.

File: source/reference/access-mdb-shell-help.txt (hunk 6 above)
Comment (davemungo) [reply source/reference/access-mdb-shell-help.txt]: Done

File: source/reference/access-mdb-shell-help.txt (hunk 1 above)
Comment (davemungo) [reply source/reference/access-mdb-shell-help.txt]: Done

File: source/reference/access-mdb-shell-help.txt (hunk 2 above)
Comment (davemungo) [reply source/reference/access-mdb-shell-help.txt]: Done

File: source/reference/access-mdb-shell-help.txt (hunk 3 above)
Comment (davemungo) [reply source/reference/access-mdb-shell-help.txt]: Done

File: source/reference/access-mdb-shell-help.txt (hunk 4 above)
Comment (davemungo) [reply source/reference/access-mdb-shell-help.txt]: Done

**Review by davemungo — COMMENTED** [review]
Thanks @jason-price-mongodb sending on to tech review next

**Review by addaleax — APPROVED** [review]

**Pull request merged**